dist/jquery.flot.js

```diff
@@ -373,7 +372,7 @@
                     }
                 }
 
-                layer.show();
+                layer.style.display = '';
```

Comment:
Not sure, I'll save the display value before hiding the div and re-apply it back.